### PR TITLE
Doc SQL Server support under CDC Source

### LINKED ISSUE #995: Document the support for sqlserver cdc via debezium
### Related code PR

the demo: https://github.com/risingwavelabs/risingwave/pull/10556

### Which part(s) of the docs might be affected or should be updated? And how?

https://www.risingwave.dev/docs/current/create-source-cdc/

### Reference

_No response_

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  Doc SQL Server support under CDC Source by referencing the SQL Server Debezium connecter under Deb JSON and AVRO. 

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/995

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/create-source/create-source-cdc.md [hunk 1]
```diff
@@ -15,17 +15,17 @@ For RisingWave to ingest CDC data, you must create a table (`CREATE TABLE`) with
 
 RisingWave accepts these data formats:
 
-- Debezium JSON (for MySQL and PostgreSQL)
+- Debezium JSON (for MySQL and PostgreSQL and SQL Server)
```
Comment (CharlieSYH) [inline docs/create-source/create-source-cdc.md]:
```suggestion
- Debezium JSON (for MySQL, PostgreSQL, and SQL Server)
```

File: docs/create-source/create-source-cdc.md [hunk 2]
```diff
@@ -15,17 +15,17 @@ For RisingWave to ingest CDC data, you must create a table (`CREATE TABLE`) with
...
+    For Debezium JSON (`FORMAT DEBEZIUM ENCODE JSON`), you can use the [Debezium connector for MySQL](https://debezium.io/documentation/reference/stable/connectors/mysql.html), [Debezium connector for PostgreSQL](https://debezium.io/documentation/reference/stable/connectors/postgresql.html), or [Debezium connector for SQL Server](https://debezium.io/documentation/reference/stable/connectors/sqlserver.html) to convert CDC data to Kafka or Pulsar topics, or Kinesis data streams.
 
 - Debezium Mongo JSON (for MongoDB)
 
     For Debezium Mongo JSON (`FORMAT DEBEZIUM_MONGO ENCODE JSON`), you can use the [Debezium connector for MongoDB](https://debezium.io/documentation/reference/stable/connectors/mongodb.html) to convert CDC data to Kafka topics.
 
-- Debezium AVRO (for MySQL and PostgreSQL)
+- Debezium AVRO (for MySQL and PostgreSQL and SQL Server)
```
Comment (CharlieSYH) [inline docs/create-source/create-source-cdc.md]:
```suggestion
- Debezium AVRO (for MySQL, PostgreSQL, and SQL Server)
```

**Review by CharlieSYH — APPROVED** [review]
Might want to apply the changes to v1.0.0. Update the branch first.

**Review by hengm3467 — APPROVED** [review]

**Comment by hengm3467** [comment]
@emile-00 Let's update the integration page to reflect the new support.

**Review by neverchanje — APPROVED** [review]
lgtm

**Pull request merged**